conf/jenkins/configuration.yaml

```diff
@@ -0,0 +1,55 @@
+jenkins:
+  agentProtocols:
+    - "JNLP4-connect"
+    - "Ping"
+  numExecutors: 0
+  clouds:
+    - mesos:
+        agentUser: "${JENKINS_AGENT_USER:-root}"
+        frameworkName: "${JENKINS_FRAMEWORK_NAME:-Jenkins Scheduler}"
+        jenkinsURL: "http://${JENKINS_FRAMEWORK_NAME:-jenkins}.${MARATHON_NAME:-marathon}.mesos:${PORT0:-8080}"
+        mesosAgentSpecTemplates:
+          - label: "${JENKINS_LINUX_AGENT_LABEL:-linux}"
+            agentAttributes: "${JENKINS_LINUX_AGENT_OFFER_SELECTION_ATTRIBUTES:-}"
+            agentCommandStyle: Linux
+            containerInfo:
+              dockerForcePullImage: false
+              dockerImage: "${JENKINS_LINUX_AGENT_IMAGE:-mesosphere/jenkins-dind:0.6.0-alpine-experimental}"
+              dockerPrivilegedMode: false
+              isDind: false
+              networking: HOST
+              type: "DOCKER"
+            cpus: ${JENKINS_LINUX_AGENT_CPUS:-0.1}
+            disk: ${JENKINS_LINUX_AGENT_DISK:-0.0}
+            domainFilterModel: "home"
+            idleTerminationMinutes: ${JENKINS_LINUX_AGENT_IDLE_TERMINATION_MINUTES:-3}
+            jnlpArgs: "${JENKINS_LINUX_AGENT_JNLP_ARGS:--noReconnect}"
+            maxExecutors: ${JENKINS_LINUX_AGENT_MAX_EXECUTORS:-1}
+            mem: ${JENKINS_LINUX_AGENT_MEM:-512}
+            minExecutors: ${JENKINS_LINUX_AGENT_MIN_EXECUTORS:-1}
+            mode: EXCLUSIVE
+          - label: "${JENKINS_WINDOWS_AGENT_LABEL:-windows}"
+            agentAttributes: "${JENKINS_WINDOWS_AGENT_OFFER_SELECTION_ATTRIBUTES:-os:windows}"
+            agentCommandStyle: Windows
+            containerInfo:
+              dockerForcePullImage: false
+              dockerImage: "${JENKINS_WINDOWS_AGENT_IMAGE:-mesosphere/jenkins-windows-node:latest}"
```

Comment:
I think we should check in the [Dockerfile](https://github.com/mesosphere/jenkins-mesos-plugin/blob/master/nodes/Dockerfile.windows) for the Windows image.

Reply:
We kept the images seperate from this repo so far: https://github.com/mesosphere/dcos-jenkins-dind-agent

Should we do the same for the Windows image if it gets complicated down the line?

Reply:
What is the benefit of this? Any new engineer will have to dig through our repo to find it.